src/adapter/src/session/vars.rs

```diff
@@ -1428,6 +1449,31 @@ impl Value for [String] {
     }
 }
 
+impl Value for Vec<String> {
+    const TYPE_NAME: &'static str = "string array";
+
+    fn parse(s: &str) -> Result<Vec<String>, ()> {
```

Comment:
I may have misunderstood, but I thought we decided against this syntax: https://github.com/MaterializeInc/materialize/issues/15208#issuecomment-1283318785?

Reply:
I opened the PR with the state from yesterday evening and only saw the updated message after that. I will have to extend the parser before we can merge this.